templates/mlops/kubeflow/index.html

```diff
@@ -38,7 +38,9 @@ <h1>
           </p>
         </div>
         <div class="p-cta-block">
-          <a href="#" class="p-button--positive js-invoke-modal" aria-controls="">Get in touch</a>
+          <a href="/contact-us"
```

Comment:
```suggestion
          <a href="/mlops/contact-us"
```

We also need the same form as static contact page, which I have added in my [PR here](https://github.com/canonical/canonical.com/pull/1844/files#diff-df971e729f4fedc9d5378280ed1d43d592632f8e19203d3fe60caf3ffdfbf4ac) so you just need to update the url here. This url will be used when javascript is disabled.